pages/pil/pdf/views/pil.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import StandardConditions from '../../../common/views/pdf/standard-conditions';
+import ReactMarkdown from 'react-markdown';
+
+const PIL = ({ pil, licenceHolder, establishment, content }) => {
+  const hasSpecies = pil.species && pil.species.length > 0;
+  const hasProcedures = pil.procedures && pil.procedures.length > 0;
+  const hasAdditionalConditions = pil.conditions && pil.conditions.length > 0;
+
+  return (
+    <Fragment>
+      <div className="logo"></div>
+
+      <h3 className="licence">PERSONAL LICENCE</h3>
+
+      <h1 className="licence-holder">{`${licenceHolder.firstName} ${licenceHolder.lastName}`}</h1>
+
+      {/* <ReactMarkdown>{content.legalPreamble}</ReactMarkdown> */}
```

Comment:
If I include the text below via `ReactMarkdown` it seems to cause the styling to break, and I'm not sure why.